voice/text-to-speech-outbound-slim/index.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use \Psr\Http\Message\ServerRequestInterface as Request;
+use \Psr\Http\Message\ResponseInterface as Response;
+use Slim\Factory\AppFactory;
+require __DIR__ . '/vendor/autoload.php';
+
+
+$app = AppFactory::create();
+
+$app->get('/webhook/answer', function (Request $request, Response $response) {
+    $client = new GuzzleHttp\Client();
+    $apiResponse = json_decode($client->get('http://api.icndb.com/jokes/random?limitTo=[nerdy]')->getBody());
+
+    $data = [
```

Comment:
In our other code snippets this variable is $ncco rather than $data. Perhaps change for consistency.

Reply:
100% for consistency, I've changed it now.